src/main/webapp/WEB-INF/tags/wrapper.tag

```diff
@@ -80,6 +91,72 @@
 </div>
 
 <header role="banner" id="global-header" class=" with-proposition ">
+
+    <div class="govuk-visually-hidden govuk-cookie-banner " data-nosnippet role="region"
+         aria-label="Reject all cookies" id="reject-all-cookies-success">
+        <div class="govuk-cookie-banner__message govuk-width-container">
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <div class="govuk-cookie-banner__content govuk-p-size-override">
+                        <p class="govuk-body govuk-p-size-override"><spring:message code="public.cookie.banner.rejected.text_0001"/></p>
+                        <p class="govuk-body govuk-p-size-override"><spring:message code="public.cookie.banner.rejected.text_0002"/> <a href="/cookie-preferences"><spring:message code="public.cookie.banner.rejected.text_0003"/></a>
```

Comment:
Do we need to explicitly set the href of the link to ${pageContext.request.contextPath}/cookie-preferences?

Reply:
Here too yes